reporting/impl/src/main/java/org/jboss/windup/reporting/rules/generation/techreport/TechReportService.java

```diff
@@ -208,7 +209,7 @@ public TechStatsMatrix getTechStatsMap(ProjectModel onlyForApplication)
             mergeToTheRightCell(map, "", placement.box.getName(), 0L, "", stat, true);
 
             List<Long> appsToCountTowards;
-            if (onlyForApplication == null)
```

Comment:
This list could be retrieved out of the loop therefore not retrieving the same list several times ?

Reply:
> I haven't seen any test modification for these changes. Are the use cases in the current existing Test covering these changes ?

The original changes caused the tests to fail - these changes fix that